tests/Fixer/Alias/ModernizeStrposFixerTest.php

```diff
@@ -27,11 +28,35 @@
  */
 final class ModernizeStrposFixerTest extends AbstractFixerTestCase
 {
+    public function testConfigure(): void
+    {
+        $this->fixer->configure(['modernize_stripos' => true]);
+
+        $reflectionProperty = new \ReflectionProperty($this->fixer, 'configuration');
+        $reflectionProperty->setAccessible(true);
+
+        self::assertSame(
+            ['modernize_stripos' => true],
+            $reflectionProperty->getValue($this->fixer)
+        );
+    }
+
+    public function testInvalidConfiguration(): void
+    {
+        $this->expectException(InvalidFixerConfigurationException::class);
+        $this->expectExceptionMessage('[modernize_strpos] Invalid configuration: The option "invalid" does not exist. Defined options are: "modernize_stripos".');
+
+        $this->fixer->configure(['invalid' => true]);
+    }
+
     /**
+     * @param array<string, mixed> $configuration
```

Comment:
after https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8019#issuecomment-2253830521,
you can do this

```suggestion
     * @param _AutogeneratedInputConfiguration $configuration
```